src/routes/reader.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const readerController = require('../controllers/reader');
+
+const router = express.Router();
+
+router.post('', readerController.addReader);
+router.get('', readerController.getAllReaders);
```

Comment:
Although there is nothing wrong with this, it is considered best practice to put a / between the quotations in the first parameter if there is no specific route after the base route
```
router.post('/', readerController.addReader);
router.get('/', readerController.getAllReaders);
```